DS13/code/modules/overmap/events.dm

```diff
@@ -351,4 +351,117 @@ GLOBAL_LIST_INIT(overmap_event_spawns, list())
 		warbird.target = target
 		warbird.linked_event = src
 		elements += warbird
-*/
\ No newline at end of file
+*/
+
+
+
+
+//----------------------
+//ITEM DELIVERY AKA SPACE UPS
+//----------------------
+
+//LANDMARK
+
+GLOBAL_DATUM(sealedcratespawn,/obj/effect/landmark/sealedcrate)
+
+/obj/effect/landmark/sealedcrate
+
+/obj/effect/landmark/sealedcrate/New()
+	. = ..()
+	world.log << "Landmark new called,setting GLOB"
+	GLOB.sealedcratespawn = src //going to personally kill you if you add more than on without refactoring this to support it
+	world.log << "Glob sealedcratespawn is now [GLOB.sealedcratespawn]"
+
+
+//CRATE
+
+/obj/structure/sealedcrate
+	name = "Sealed crate"
+	desc = "Very important crate that contains very important stuff that starfleet holds dear to them. \
+			Better not mess this up or you are going to end up scrubbing plasma conduits."
+	icon = 'icons/obj/crates.dmi'
+	icon_state = "sealedcrate"
+	var/datum/overmap_event/linked_event
+	max_integrity = 120
+	density = TRUE
+
+/obj/structure/sealedcrate/attack_hand(mob/user)
+	. = ..()
+	if(.)
+		return
+	to_chat(user,"<span class='warning'>This crate is locked and sealed,you can't open it.</span>") //HEY LEAVE THIS ALONE OR YOUR SCRUBBING PLASMA CONDUITS
+
+/obj/structure/sealedcrate/Destroy()
+	. = ..()
+	linked_event.fail()
+	linked_event = null
+
+//DESTINATION_PORTAL
+
+/obj/structure/crate_receiver
+	name = "Delivery point"
+	desc = "Deliver sealed crate here"
+	icon = 'DS13/icons/obj/cratereceiver.dmi'
+	icon_state = "delivery_point"
+	density = FALSE
+	opacity = FALSE
+
+/obj/structure/crate_receiver/Crossed(atom/movable/AM, oldloc)
+	. = ..()
+	if(istype(AM,/obj/structure/sealedcrate))
+		var/obj/structure/sealedcrate/SC = AM
+		SC.linked_event.succeed()
```

Comment:
if(SC.linked_event)

Reply:
It shouldn't ever exist without a linked_event.